lib/Microservice.js

```diff
@@ -0,0 +1,141 @@
+const loopback = require('loopback');
+const boot = require('loopback-boot');
+
+
+/**
+ * Basic Microservice class wrapping a loopback application.
+ *
+ * @type {module.Microservice}
+ */
+
+const Microservice = module.exports = class Microservice {
```

Comment:
Is there a reason to assign this to a local const?

Reply:
Just a habit... will change it.